doc/osi.sgml

```diff
@@ -161,6 +161,42 @@ There is no specific support for direct hardware access.
 
 There are no loadable drivers available.
 
+<sect>Support for different screen layouts<p>
+
+By default the conio library uses a 24 columns by 24 lines screen layout
+for the Challenger 1P, like under BASIC. In addition to that there is support
+for other screen layouts with extra configuration modules.
+
+There is a default module <tt/screen.s/ in the OSI-specific
+cc65 runtime library. This module contains all parameters
+related to the screen layout. No further configuration is needed for using the
+default screen layout of the Challenger 1P.
+
+For other screen layouts additional copies of the screen.s module must
+be created in the <tt>libsrc/osic1p/extra</tt> directory. The cc65 build
```

Comment:
The documentation at hand is targeted towared the cc65 user (aka the C or ASM programmer using it). However what is described here is relevant for the cc65 developer only. So it doesn't belong here.